docker/worker/kvm-mknod.sh

```diff
@@ -3,7 +3,7 @@
 
 set -e
 
-lsmod | grep '\<kvm\>' > /dev/null || {
+test $(cat /proc/config.gz | gunzip | grep CONFIG_KVM=y) || lsmod | grep '\<kvm\>' > /dev/null || {
```

Comment:
Did you know that

a) The Useless Use of Cat Award is a thing (http://porkmail.org/era/unix/award.html)?
b) that gunzip has a -c option?

Reply:
No, I did not. :)
Fixed.